src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileUploadsValidatorService.cs

```diff
@@ -48,7 +48,8 @@ public FileUploadsValidatorService(ISubjectService subjectService, IFileTypeServ
                 .OnSuccess(async _ => await ValidateSubjectName(releaseId, name));
         }
 
-        public async Task<Either<ActionResult, Unit>> ValidateFileForUpload(Guid releaseId, IFormFile file, ReleaseFileTypes type, bool overwrite)
+        public async Task<Either<ActionResult, Unit>> ValidateFileForUpload(Guid releaseId, IFormFile file,
+            ReleaseFileTypes type, bool overwrite, string? name = "")
```

Comment:
Name can be nullable?

Reply:
OK - have added a separate method to validate the file upload name as probably clearer that way.